rpa-manager/modules/ROOT/nav.adoc

```diff
@@ -1,7 +1,12 @@
 .xref:index.adoc[RPA Manager]
 * xref:index.adoc[RPA Manager Overview]
 * xref:organizationmanagement-overview.adoc[Organization Management]
-** xref:organizationmanagement-settings.adoc[Configure Organization Settings]
+** xref:organizationmanagement-settings.adoc[Configuring Organization Settings]
```

Comment:
We try not to have a single bulleted item.

Reply:
Leaving this for now. I will rework the navigation for the next release.